src/main/java/jpa/dao/SubcategoryDao.java

```diff
@@ -0,0 +1,54 @@
+package jpa.dao;
+
+import tecnodev.subCategory.SubCategory;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+
+public class SubcategoryDao {
+
+    private EntityManager em;
+
+    public SubcategoryDao(EntityManager em) {
+        this.em = em;
+    }
+
+    public List<SubCategory> getSubcategory() {
```

Comment:
Que tal um nome mais semântico no sentido de trazer todas as subcategorias?

```suggestion
    public List<SubCategory> getAllSubcategories() {
```

Reply:
Dei uma olhada e na verdade esse método não está sendo usado em nenhum local, acho que irei apaga-lo. Eu devo ter feito algum teste no inicio da semana e ele ficou ali.